### PR TITLE
Feature/fallout tests with GKE

### PR DESCRIPTION
**Pull request by marianne-manaog**
The aim of this PR is to add six Fallout-related performance tests that can be run via the open-source Fallout, assuming that GKE is available and used to run the tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
-# fallout-tests
+# Fallout tests with GKE
+
+
+## What is Fallout
+[Fallout](https://github.com/datastax/fallout) is a tool for running local or large scale remote-based distributed correctness, verification and performance tests. Fallout is run in production at DataStax facilitating mission critical testing of Apache Cassandra (TM) and Apache Pulsar (TM).
```
Comment (mfleming) [inline README.md]: The tool runs locally but the test actually runs remotely.
```suggestion
[Fallout](https://github.com/datastax/fallout) is a tool for running large scale remote-based distributed correctness, verification and performance tests for Apache Cassandra (TM) and Apache Pulsar (TM).
```

File: README.md [hunk 2]
```diff
@@ -1 +1,40 @@
...
+
+Fallout uses Kubernetes and the Helm Chart to deploy the services and run the tests. The size of the cluster is limited only by the available resources.
+## What this repository is for
+This repository enables to run performance tests on Cassandra
```
Comment (mfleming) [inline README.md]:
```suggestion
This repository contains performance tests for Cassandra 
```

File: template-params.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+image:
```
Comment (mfleming) [inline template-params.yaml]: The contents of this file are unused so I think you can delete everything here and add this:
```suggestion
gke_project: SET_PROJECT_ID
```
and then delete `gke_project` from all the test files in `lwt/*.yaml`.

**Review by mfleming — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (marianne-manaog) [reply README.md]: Thanks - Addressed [here](https://github.com/datastax/fallout-tests/pull/1/commits/34aeadde129de41d6fd8332984d0506941875d0d)

File: README.md (hunk 2 above)
Comment (marianne-manaog) [reply README.md]: Thanks - Addressed [here](https://github.com/datastax/fallout-tests/pull/1/commits/14ee404046ba89ab46ef084c9bbfb471a7b4694f)

File: template-params.yaml (hunk 3 above)
Comment (marianne-manaog) [reply template-params.yaml]: Thanks - Addressed [here](https://github.com/datastax/fallout-tests/pull/1/commits/65a8b13f5e50729b143a9e3db55fe0dfccdbf035) and [here](https://github.com/datastax/fallout-tests/pull/1/commits/2877d57078c16b29dc8f60a439f61e44b37caf07)

**Review by mfleming — APPROVED** [review]
Looks good!

**Pull request merged**